development/program.ttl

```diff
@@ -487,6 +527,28 @@ hpc:endMemoryAddress rdf:type owl:DatatypeProperty ;
                      rdfs:subPropertyOf hpc:memoryAddress .
 
 
+###  https://hpc-fair.github.io/ontology#exclusivePercentageTime
+hpc:exclusivePercentageTime rdf:type owl:DatatypeProperty ;
+                            rdfs:range xsd:decimal ,
+                                       xsd:double ,
+                                       xsd:float ;
+                            rdfs:comment "The proportion (percentage) of the execution time of the program spent executing this function, excluding time spent executing callee functions." .
+
+
+###  https://hpc-fair.github.io/ontology#exclusiveTime
+hpc:exclusiveTime rdf:type owl:DatatypeProperty ;
+                  rdfs:range xsd:decimal ,
+                             xsd:double ,
+                             xsd:float ;
+                  rdfs:comment "Time spent in a function, excluding time spent executing children (callee) functions" .
+
+
+###  https://hpc-fair.github.io/ontology#hasProfiling
+hpc:hasProfiling rdf:type owl:DatatypeProperty ;
```

Comment:
This looks a bit redundant. A proper way to capture this kind of info. is to have a OWL property hierarchy of static vs. dynamic/profiling information for a function. Then write queries to query this information (if a function has profiling properties).  Otherwise, someone has to manually update this bool value if a profiling information is added for a function.

Reply:
This is a good point; I will remove it in the upcoming commit